test/integration/api/post/index.test.js

```diff
@@ -0,0 +1,87 @@
+import test from 'ava';
+import request from 'supertest-as-promised';
+import { db } from '../../../../src/services';
+import { schemas } from '../../../fixtures';
+
+const url = process.env.API_URL;
+const auth = process.env.AUTH_URL;
+
+let token;
+
+test.before('user:Signup', async t => {
+  await db(schemas, process.env.RETHINKDB_URL, process.env.RETHINKDB_NAME).start();
+  t.truthy(db().instance().conn, 'connection is present');
+
+  const table = await db().instance().clearTable('user');
+
+  t.is(table, true, 'person table successfully cleared');
+
+  const signup = await request(auth)
+     .post('/signup')
+     .set('content-type', 'application/json')
+     .send(JSON.stringify({
+       name: 'CJ',
+       role: '1',
+       email: 'cj@api.com',
+       password: `${process.env.INTEGRATION_PASSWORD}`,
+     }));
+
```

Comment:
Those chained methods are indented too much. Same with some of the functions below.